language-plutus-core/bench/Bench.hs

```diff
@@ -1,13 +1,22 @@
+{-# LANGUAGE OverloadedStrings #-}
+
 module Main (main) where
 
 import           Codec.Serialise
 import           Control.Monad
 import           Criterion.Main
-import qualified Data.ByteString.Lazy       as BSL
-import qualified Data.Text                  as T
+import           Crypto
+import qualified Data.ByteString.Lazy                     as BSL
+import qualified Data.Text                                as T
 import           Language.PlutusCore
+import           Language.PlutusCore.Evaluation.CkMachine (runCk)
 import           Language.PlutusCore.Pretty
 
+pubKey, sig, msg :: BSL.ByteString
+sig = "e5564300c360ac729086e2cc806e828a84877f1eb8e5d974d873e065224901555fb8821590a33bacc61e39701cf9b46bd25bf5f0595bbe24655141438e7a100b"
+pubKey = "d75a980182b10ab7d54bfed3c964073a0ee172f3daa62325af021a68f707511a"
+msg = ""
```

Comment:
I'd really prefer to use the appropriate functions to generate the signatures where possible rather than using hardcoded values.

Reply:
Unfortunately, `cardano-crypto` does not include a function to generate keys.

Reply:
Well, we're going to need that.

Presumably at least it has a function to sign things?

Reply:
Yes, it does. We can ask upstream at `cardano-crypto` for such a feature.

Reply:
Also, for whatever reason, `sign` in `cardano-crypto` takes salt but `verify` does not.